dataland-frontend/src/components/general/TheQuotes.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <section v-if="quotesSection" class="quotes" role="region" aria-label="The Quotes">
+    <SlideShow
+      slides-container-classes="quotes__slides"
+      arrows-container-classes="quotes__arrows"
+      left-arrow-classes="quotes__arrow quotes__arrow--left"
+      right-arrow-classes="quotes__arrow quotes__arrow--right"
+      :slide-count="slides.length"
+      :initial-center-slide="1"
+      @update:currentSlide="(newSlide) => (currentSlide = newSlide)"
+    >
+      <div v-for="(card, index) in cards" :key="index" role="listitem" class="quotes__slide">
+        <div class="quotes__slide-videoContainer">
+          <iframe
+            :src="'https://www.youtube.com/embed/' + card.icon + '?rel=0'"
+            title="Youtube video player"
+            allowfullscreen
+            class="quotes__slide-video"
+          ></iframe>
+        </div>
+        <h3 class="quotes__slide-title">
+          {{ card.title }} <span>{{ card.text }}</span>
+        </h3>
+      </div>
+    </SlideShow>
+    <p class="quotes__slide-text">{{ currentCardText }}</p>
+    <register-button :buttonText="quotesSection.text[0]" />
```

Comment:
Can we spell it "RegisterButton" here so my IDE can find its usage via the search function?

Reply:
Yes, I change it